web_client/src/components/ExperimentsView.vue

```diff
@@ -280,6 +280,13 @@ export default {
           </li>
         </ul>
       </div>
+      <div
+        v-else-if="currentProject.experiments.length"
```

Comment:
Shouldn't we use an out-of-band `loading: boolean` state for this? What if the project simply has no experiments yet?

Reply:
Oh, I see. Thanks for the explanation.